### PR TITLE
Notifications outside of page context

### PR DESCRIPTION
**Pull request by pxlrbt**
I was trying to wrap my head around how to get notifications to work with ButtonActions. Feel free to hit me up on Discord if you want to discuss this or let me know what you think.

### CODE REVIEW

File: packages/admin/src/Pages/Page.php [hunk 1]
```diff
@@ -73,14 +78,6 @@ public static function getUrl(array $parameters = [], bool $absolute = true): st
         return route(static::getRouteName(), $parameters, $absolute);
     }
 
-    protected function notify(string $status, string $message): void
```
Comment (ryangjchandler) [inline packages/admin/src/Pages/Page.php:76]: Removing this method is a breaking change.

**Review by ryangjchandler — CHANGES_REQUESTED** [review]

**Comment by ryangjchandler** [comment]
I think there might be a better way to do this...

@danharrin Could we not refresh the `Page` component with an event listener instead, i.e. emit an event when an action is finished and then:

```php
protected $listeners = [
    'someActionCompleteEvent' => '$refresh',
];
```

This would then trigger a re-render and the notification will appear.

File: packages/admin/src/Pages/Page.php (hunk 1 above)
Comment (pxlrbt) [reply packages/admin/src/Pages/Page.php:76]: It extracted into a Trait.

Comment (ryangjchandler) [reply packages/admin/src/Pages/Page.php:76]: Ah, yeah. Had that files collapsed for some reason.

**Comment by pxlrbt** [comment]
> My reasoning behind this is that the Page component shouldn't really hold any state regarding notifications. Having the method to notify() is fine, but actually holding those notifications on the component feels a bit iffy.

Yeah, I felt the same. Ideally we would have a NotificationManager. I'm still quite new to Livewire so I didn't know how to approach this otherwise.

**Comment by danharrin** [comment]
Yeah, I don't think this is the right solution. I think we should have a `NotificationManager` component which we then emit events to to add notifications. This could also support multiple stacked notifications at a time maybe? I'll add this to my low priority todos, but if you have chance I'd appreciate a PR (if you want to attempt this).

**Pull request closed without merging**